internal/util/invalid_serialization_detector.go

```diff
@@ -11,21 +11,27 @@ func NewInvalidSerializationDetectorSchema() *InvalidSerializationDetectorSchema
 	return &InvalidSerializationDetectorSchema{}
 }
 
-// InvalidSerializationDetectorSchema is a testing type that detects double serialization or double unserialization,
-// which helps prevent bugs.
-// It is backed by a string, which is set to the last operation performed. If the last operation matches the
-// current operation, then that indicates double serialization or double unserialization.
-type InvalidSerializationDetectorSchema struct{}
+// InvalidSerializationDetectorSchema is a testing type that detects double-
+// serialization or double-unserialization which could result in corrupted data.
+// The serialization and unserialization methods detect when the operation is
+// performed twice in a row on a single piece of data by examining the input
+// value; they also count the number of overall operations so that we know that
+// the data has been serialized or unserialized at least once (since, if it is
+// never operated on, then it's trivial to claim that it was never doubly done).
```

Comment:
trivial and entirely accurate; merely useless. 😆 

But, "doubly done"? Umm, hmm...

Reply:
Yeah, I got tired of typing "doublely-serialized and/or double-unserialized"...so I started talking about "operations", and then even that became cumbersome....

Alternative wordings would be welcome....  🙂

Reply:
I think it would be better to just drop the parenthesized phrase entirely: I don't think the awkwardly worded "clarification" provides any real value.

Reply:
Sorry, Dave!  I didn't see this until just now (long after the merges).  Hopefully, it doesn't actually matter.